Index: fusiondirectory-1.0.2/contrib/bin/fusiondirectory-setup
===================================================================
--- fusiondirectory-1.0.2.orig/contrib/bin/fusiondirectory-setup	2011-10-16 23:18:50.000000000 +0200
+++ fusiondirectory-1.0.2/contrib/bin/fusiondirectory-setup	2011-10-16 23:23:21.000000000 +0200
@@ -44,7 +44,10 @@
 
 my $yes_flag = 0;
 
-##################################################################################################################################################
+my %classes_hash_result = ();
+my %i18n_hash_result = ();
+
+#################################################################################################################################################
 
 #die if the user is not root
 die ("! You have to run this script as root\n") if ($<!=0);
@@ -78,7 +81,6 @@
 sub get_classes {
 
     my ($path) = @_;
-    my %hash_result = ();
 	
     # if this function has been called without a parameter
     die ("! function get_classes called without parameter\n") if ( !defined($path) );
@@ -92,7 +94,7 @@
     foreach my $file (@dir_files) {
         # recursive call if $file is a directory
         if ( -d $file ) {
-            %hash_result = (%hash_result, get_classes($file));
+            %classes_hash_result = get_classes($file);
             next;
         }
 
@@ -111,12 +113,12 @@
 						# only process for lines beginning with "class", and extracting the 2nd word (the class name)
 						if ( $line =~ /^class\s*(\w+).*/ ) {
 							# adding the values (class name and file path) to the hash
-							$hash_result{$1} = $file;
+							$classes_hash_result{$1} = $file;
 						}
           }
         }
     }
-	return %hash_result;
+	return %classes_hash_result;
 }
 
 # call get_classes and create /var/cache/fusiondirectory/class.cache
@@ -151,7 +153,6 @@
 sub get_i18n {
 
     my ($path) = @_;
-    my %hash_result = ();
 	
     # if this function has been called without a parameter
     die ("! function get_i18n called without parameter" ) if ( !defined($path) );
@@ -165,17 +166,17 @@
     foreach my $file (@dir_files) {
 			# recursive call if $file is a directory
 			if (-d $file) {
-				%hash_result = ( %hash_result, get_i18n ($file) );
+				%i18n_hash_result = get_i18n ($file);
 				next;
 			}
 
 			# if the file's directory is ???/language/LC_MESSAGES/messages.po
 			if ($file =~ qr{^.*/(\w+)/LC_MESSAGES/messages.po$}) {
 				# push the file's path in the language (fr/en/es/it...) array (wich is inside the hash pointed by $ref_result
-				push @{$hash_result{$1}}, $file;
+				push @{$i18n_hash_result{$1}}, $file;
 			}
     }
-	return %hash_result;
+	return %i18n_hash_result;
 }
 
 # call get_i18n with the FusionDirectory's locales's directory and the hash that will contain the result in parameter
