--- a/contrib/bin/fusiondirectory-setup
+++ b/contrib/bin/fusiondirectory-setup
@@ -31,7 +31,7 @@
 
 # fd's directory and class.cache file's path declaration
 my %vars = (
- fd_home          => "/var/www/fusiondirectory",
+ fd_home          => "/usr/share/fusiondirectory",
  fd_cache         => "/var/cache/fusiondirectory",
  fd_config_dir    => "/etc/fusiondirectory",
  fd_smarty_dir    => "/usr/share/php/smarty3",
